storage/internal/benchmarks/README.md

```diff
@@ -19,7 +19,7 @@ This runs 1000 iterations on 512kib to 2Gib files in the background, sending out
 | -samples | number of samples to report | any positive integer | `8000` |
 | -workers | number of goroutines to run at once; set to 1 for no concurrency | any positive integer | `16` |
 | -clients | total number of Storage clients to be used; <br> if Mixed APIs, then x3 the number are created | any positive integer | `1` |
-| -api | which API to use | `JSON`: use JSON to upload and XML to download <br> `XML`: use JSON to upload and XML to download <br> `GRPC`: use GRPC <br> `Mixed`: select an API at random for each upload/download  <br> `DirectPath`: use GRPC with direct path | `Mixed` |
+| -api | which API to use | `JSON`: use JSON <br> `XML`: use JSON to upload and XML to download <br> `GRPC`: use GRPC <br> `Mixed`: select an API at random for each object <br> `DirectPath`: use GRPC with direct path | `Mixed` |
```

Comment:
thanks for updating, maybe just specify as well that GRPC implies no directpath.

Reply:
done!